packages/teleport/src/Audit/__snapshots__/Audit.story.test.tsx.snap

```diff
@@ -4103,43 +4140,6 @@ exports[`list of all events 1`] = `
           </button>
         </td>
       </tr>
-      <tr>
```

Comment:
Unrelated to app access events, but the tests were failing because this event seems to be missing from fixtures so I removed it from here as well.